README.md

```diff
@@ -1975,7 +1993,7 @@ Note that there are `dataSource` configurations, that go under `dataSource`, lik
 
 The `JdbcContext` provides thread-local transaction support:
 
-```scala
+```
```

Comment:
Why not indicate this is scala code?

Reply:
because it fails the to compile with tut